app/src/main/java/org/zalando/nakadi/config/WebConfig.java

```diff
@@ -75,6 +76,12 @@ public TimeoutCallableProcessingInterceptor timeoutInterceptor() {
         return new TimeoutCallableProcessingInterceptor();
     }
 
+
+    // ==========================================================================================
```

Comment:
does it mean that all the filters in the section are called before authentication token is validated?

Reply:
I suggest to put every nakadi filter after spring security filters

Reply:
> does it mean that all the filters in the section are called before authentication token is validated?

Yes, I understand it that way as well.

> I suggest to put every nakadi filter after spring security filters.

I think this is undesirable, as we might want to trace calls to the token info, and for that it is important that the tracing span is established earlier.

It also seems that `RequestRejectedFilter` needs a higher precedence to achieve the desired effect.

Reply:
I understand, that it is current order of filters, then lets leave as it is